subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -60,11 +60,39 @@ void radio_reset(void)
 	     RADIO_POWER_POWER_Msk);
 }
 
-void radio_phy_set(u8_t phy)
+void radio_phy_set(u8_t phy, u8_t flags)
 {
-	NRF_RADIO->MODE =
-	    (((phy) ? (u32_t)phy : RADIO_MODE_MODE_Ble_1Mbit) <<
-	     RADIO_MODE_MODE_Pos) & RADIO_MODE_MODE_Msk;
+	u32_t mode;
+
+	switch (phy) {
+	case BIT(0):
```

Comment:
I will let this as-is as radio driver needs rework; define a standard across vendors later. https://jira.zephyrproject.org/browse/ZEP-731